Alphafold.dockerfile

```diff
@@ -57,16 +56,16 @@ ENV PATH="/opt/conda/bin:$PATH"
 ##  Bug-Fix: Running the conda version 4.13.0, we face an incompatibility issue with python 3.11. 
 ##  Issue and fix appled as suggested in this github issue: https://github.com/deepmind/alphafold/issues/798
 #RUN conda install -qy conda==4.13.0 \
-RUN conda install -qy conda==23.1.0 \
+RUN conda install -qy conda==24.1.2 \
     && conda install -y -c conda-forge \
-      openmm=7.5.1 \
+      openmm=7.7.0 \
       pdbfixer \
       pip \
-      python=3.9 \
+      python=3.10 \
       && conda install -y -c nvidia/label/cuda-${CUDA} cuda-libraries-dev cuda-nvcc cuda-nvtx cuda-cupti \
       && conda clean --all --force-pkgs-dirs --yes
 
-RUN git clone --branch $ALPHAFOLD_VERSION https://github.com/deepmind/alphafold.git /app/alphafold
```

Comment:
By using the latest version deployed in `main` we add a risk of a breaking change on the alphafold.git repo to impact our pipelines.

Ideally, I would recommend reseting HEAD to a specific commit you've tested out and validated that it works well.

e.g.: add git reset --hard fcbd92b

Reply:
Thx a lot for the comment, @chmstimoteo . I have updated both docker files as per your suggestion. Please check it out.